src/program.js

```diff
@@ -568,7 +568,7 @@ Example: $0 --help run.
         'amo-base-url': {
           describe:
             'Signing API URL prefix - only used with `use-submission-api`',
-          default: 'https://addons.mozilla.org/api/v5',
+          default: 'https://addons.mozilla.org/api/v5/',
```

Comment:
Did we have coverage for this URL/value? I see we're updating tests below but I don't get why nothing failed before. Maybe the default value in tests should be coming from this file instead of having yet another value that might or might not resemble this one here.

Reply:
Nothing failed because the the default value was defined 2 files (and numerous calls/processing) further up from where it was used.

Relying on locally defined defaults in unit tests rather than importing other files seemed to be the normal style of writing tests but it could be changed I guess.

Reply:
https://github.com/mozilla/web-ext/pull/2621/commits/f8cfb9f8fda5b9940c097c1e64d107202ee53af9